app/assets/stylesheets/content/_tooltips.md

```diff
@@ -107,6 +107,18 @@ These can contain simple texts but are not suitable for HTML within the Tooltip.
 
 Note that the tabindex has to be set manually on the `<span>` and not the containing element. `tabindex="0"` makes the item tabbable at all.
 
+### HTML tooltips
+
+```
+<span class="advanced-tooltip-trigger" data-tooltip-target="#advanced-tooltip-sample">
+  <i class="icon icon-help1"></i>
```

Comment:
I don't seem to get the tooltip when hovering here (like for the simple tooltips above)...